code/modules/food_and_drinks/food/snacks_pizza.dm

```diff
@@ -160,8 +160,59 @@
 	tastes = list("crust" = 1, "tomato" = 1, "cheese" = 1, "pineapple" = 2, "ham" = 2)
 	foodtype = GRAIN | VEGETABLES | DAIRY | MEAT | FRUIT | PINEAPPLE
 
+/obj/item/reagent_containers/food/snacks/pizza/arnold
+	name = "arnold pizza"
```

Comment:
capitalize the name, please.

Reply:
we dont capitalize items

Reply:
its a persons name, those are capitalized in the middle of sentences.

Reply:
`/obj/item.+\r\n\tname = "[A-Z]` - 593 hits
`^/mob.+\r\n\tname = "[A-Z]` - 80 hits

Reply:
@Jordie0608 even then they're improper so they show up as lowercase